falcon/asgi/request.py

```diff
@@ -693,16 +695,31 @@ async def get_media(self):
 
         See also :ref:`media` for more information regarding media handling.
 
+        Note:
+            When ``get_media`` is called on a request with an empty body,
+            falcon will let the media handler try to deserialize the body
+            and will return the value returned by the handler or propagate
+            the exception raised by it. To instead return a different value
+            in case of an exception by the handler, specify the argument
+            ``when_empty_fallback``.
+
         Warning:
             This operation will consume the request stream the first time
             it's called and cache the results. Follow-up calls will just
             retrieve a cached version of the object.
 
+        Args:
+            when_empty_fallback: Fallback value to return when there is no body
+                in the request and the media handler raises an error
+                (like in the case of the default JSON media handler).
+                By default falcon use the value returned by the media handler
```

Comment:
Not critical, but I would capitalize Falcon everywhere unless referring to the module name. I am aware the current docs are a bit of a mix tho.